specs/windows-terminservice-gui.spec

```
@@ -6,8 +6,8 @@ block_cipher = None
 
 a = Analysis(['..\\gui.py'],
              pathex=['specs//'],
-             binaries=[('..\\tools\\chromedriver\\chromedriver-windows.exe', 'tools\\chromedriver\\'), ('..\\tools\\gui\\kontaktdaten.ui', 'tools\\gui\\'), ('..\\tools\\gui\\main.ui', 'tools\\gui\\'), ('..\\tools\\gui\\terminsuche.ui', 'tools\\gui\\'), ('..\\tools\\gui\\impfzentren.ui', 'tools\\gui\\'),('..\\images\\spritze.ico', 'images\\')],
-             datas=[('../tools/cloudscraper', './cloudscraper/')],
+             binaries=[('..\\tools\\chromedriver\\chromedriver-windows.exe', 'tools\\chromedriver\\'), ('..\\tools\\gui\\kontaktdaten.ui', 'tools\\gui\\'), ('..\\tools\\gui\\main.ui', 'tools\\gui\\'), ('..\\tools\\gui\\terminsuche.ui', 'tools\\gui\\'), ('..\\tools\\gui\\uhrzeiten.ui', 'tools\\gui\\'), ('..\\tools\\gui\\impfzentren.ui', 'tools\\gui\\'),('..\\images\\spritze.ico', 'images\\')],
```

`tools/gui/uhrzeiten.ui` wird nicht benötigt -> Datei kann also auch komplett gelöscht werden

```suggestion
             binaries=[('..\\tools\\chromedriver\\chromedriver-windows.exe', 'tools\\chromedriver\\'), ('..\\tools\\gui\\kontaktdaten.ui', 'tools\\gui\\'), ('..\\tools\\gui\\main.ui', 'tools\\gui\\'), ('..\\tools\\gui\\terminsuche.ui', 'tools\\gui\\'), ('..\\tools\\gui\\impfzentren.ui', 'tools\\gui\\'),('..\\images\\spritze.ico', 'images\\')],
```

---

Richtig!